Bowling game menu and test

IscJacobo19 opened the pull request:
# *The menu was created with a while
 # *I added test in a different file
 # *I added a file for my first class called Game.rb

IscJacobo19 committed:
Bowling game menu and test
  *The menu was created with a while
  *I added test in a different file
  *I added a file for my first class called Game.rb

EsmeSantoyo commented on bowling_game.rb:
```diff
@@ -0,0 +1,40 @@
+    when 5..10
+      print "Option not valid. bye\n"    
+  end
+end
```
here
```suggestion
end

```

EsmeSantoyo commented on bowling_game.rb:
```diff
@@ -0,0 +1,40 @@
+    when 5..10
+      print "Option not valid. bye\n"    
+  end
+end
```
here 
```suggestion
end

```

IscJacobo19 replied:
ok. thanks you!

EsmeSantoyo commented on games.rb:
```diff
@@ -0,0 +1,5 @@
+  def balls(bowling)
+    @bowling = bowling
+  end
+end
```
here
```suggestion
end

```

EsmeSantoyo commented on test_bowling_game.rb:
```diff
@@ -0,0 +1,10 @@
+      assert_equal [20,37,44,74,104,134,164,194,224,254] ,games.balls([10, 0, 10, 0, 2, 5, 10, 0, 10, 0, 10, 0, 10, 0, 10, 0, 10, 0, 10, 10, 10])
+      assert_equal [5,14,29,49,60,61,77,97,117,133] ,games.balls([1, 4, 4, 5, 6, 4, 5, 5, 10, 0, 0, 1, 7, 3, 6, 4, 10, 0, 2, 8, 6])
+    end
+  end
```
and here
```suggestion
  end
  
```

IscJacobo19 replied:
@cfcortes  @EsmeSantoyo  Buen día, hice un comimit para corregir estos detalles, cuando se que ya está aprobado, para continuar programando?

EsmeSantoyo reviewed (changes requested):
You should add a final blank line and use rubocop to review your ruby syntax to avoid that kind of "errors".

ManuelJacobo committed:
Blank lines added
 *in the file bowling_game.rb, a empaty line was added
 *in the file games.rb, a empaty line was added
 *in the file test_bowling_game.rb a empaty line was added

cfcortes commented:
Buen punto lo de rubocop @EsmeSantoyo lo voy a agregar en las recomendaciones

IscJacobo19 committed:
Add blank lines
